build/Dockerfile

```diff
@@ -1,4 +1,71 @@
+# build stage for building binaries
+FROM registry.access.redhat.com/ubi8/go-toolset:1.12.8-32 as build
+LABEL stage=build
+
+# Build WMCB
+# WORKDIR is not resulting in creating this directory, so we are forcing it to happen. The image does not allow us to
+# create /build/, forcing us to use $HOME which /opt/app-root/src/
+RUN mkdir $HOME/build/
+WORKDIR $HOME/build/
+RUN git clone https://github.com/openshift/windows-machine-config-bootstrapper.git
```

Comment:
I thought we're go to cut a release and get all the binaries from a location or do you have any other plans?

Reply:
I want to move away from having to cut releases. Given we are going to release an operator image per OpenShift release, this would always ensure we are getting the latest and correct version.